minidump-processor/src/process_state.rs

```diff
@@ -416,6 +422,37 @@ impl PossibleBitFlip {
     }
 }
 
+#[derive(Debug, Clone)]
+pub enum CrashReasonInconsistency {
+    IntDivByZeroNotPossible,
+    PrivInstructionCrashWithoutPrivInstruction,
+    NonCanonicalAddressFalselyReported,
+    AccessViolationWhenAccessAllowed,
+    CrashingAccessNotFoundInMemoryAccesses,
+}
+
+impl std::fmt::Display for CrashReasonInconsistency {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            CrashReasonInconsistency::IntDivByZeroNotPossible => {
+                f.write_str("Crash reason is integer division by zero but instruction is not")
```

Comment:
"Crash reason is an integer division by zero but the crashing instruction is not a division"